src/oci.c

```diff
@@ -1070,7 +1050,26 @@ cc_oci_start (struct cc_oci_config *config,
 	              config->state.state_file_path, false);
 
 	if (wait) {
-		g_main_loop_run (data.loop);
+		/* connecting to hypervisor socket */
+		if (data.loop) {
+			g_main_loop_run (data.loop);
```

Comment:
I think we only need to use the main loop to wait for the hypervisor socket in "create", here is not  required anymore :) given that the  qemu process is already  running from "create" and is ready (even already talked with hyperstart on create). 
Probably should be enough just to  check a hypervisor socket exist to verify the vm is running